apps/linkedin-to-notion/src/contents/linkedin-notion-side-panel.tsx

```diff
@@ -0,0 +1,38 @@
+import cssText from 'data-text:~style.css';
+import type { PlasmoCSConfig, PlasmoGetStyle } from 'plasmo';
+import { useState } from 'react';
+
+import { LinkedInNotionSidePanelContent } from '../components/LinkedInNotionSidePanel/LinkedInNotionSidePanelContent';
+
+export const config: PlasmoCSConfig = {
+  matches: ['https://www.linkedin.com/in/*'],
+  all_frames: false,
+};
+
+// Needed for the style directly applied to the iFramedSidePanel
+export const getStyle: PlasmoGetStyle = () => {
+  const style = document.createElement('style');
+  style.textContent = cssText;
+  return style;
+};
```

Comment:
Correct me if I'm wrong, this is where the *outer* styling of the iframe happens, right? For instance, that's where you say that you want the side panel to be 30vw, or whatever. Am I correct?

Reply:
Yep!